report/test/renderer/report-renderer-axe-test.js

```diff
@@ -66,16 +66,6 @@ describe('ReportRendererAxe', () => {
       //   id: 'color-contrast',
       // },
 
-      expect(axeResults.violations.find(v => v.id === 'duplicate-id')).toMatchObject({
```

Comment:
`duplicate-id` check has been deprecated, and will likely be removed in a future axe release: https://github.com/dequelabs/axe-core/issues/3892

I don't think we need to enable it to keep this check since we aren't going to "fix" this ever.